cmd/private-org-sync/main.go

```diff
@@ -421,6 +421,9 @@ func addGitRemote(logger *logrus.Entry, git gitFunc, org, repo, repoDir, remoteN
 		remoteSetupLogger.WithError(err).Error("Failed to construct URL for the source remote")
 		return fmt.Errorf("failed to construct URL for the source remote")
 	}
+	if token != "" {
+		srcURL.User = url.User(token)
```

Comment:
Just to make sure, do we censor this?

Reply:
We should, there is the censoring logger setup, and we are already using tokens in the destination remotes.

But I'll explicitly check this new usage, better to be sure about this.

/hold